ios/RNCNetInfo.m

```diff
@@ -124,9 +130,15 @@ - (NSMutableDictionary *)detailsFromInterface:(nonnull NSString *)requestedInter
   } else if ([requestedInterface isEqualToString: RNCConnectionTypeWifi] || [requestedInterface isEqualToString: RNCConnectionTypeEthernet]) {
     details[@"ipAddress"] = [self ipAddress] ?: NSNull.null;
     details[@"subnet"] = [self subnet] ?: NSNull.null;
-    #if !TARGET_OS_TV && !TARGET_OS_OSX
-    details[@"ssid"] = [self ssid] ?: NSNull.null;
-    details[@"bssid"] = [self bssid] ?: NSNull.null;
+    #if !TARGET_OS_TV && !TARGET_OS_OSX   
+      /*
+        Without one of the conditions needed to use CNCopyCurrentNetworkInfo, it will leak.
+        Clients must set the shouldFetchWiFiSSID after ensuring one of these is met to get (B)SSID.
```

Comment:
```suggestion
        Without one of the conditions needed to use CNCopyCurrentNetworkInfo, it will leak memory.
        Clients should only set the shouldFetchWiFiSSID to true after ensuring requirements are met to get (B)SSID.
```